composer.json

```diff
@@ -13,7 +13,9 @@
     "require": {
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.7|3.*",
-        "symfony-cmf/resource": "^1.0"
+        "symfony/options-resolver": "^2.7|3.*",
+        "symfony-cmf/resource": "dev-admin_metadata",
+        "puli/repository": "@beta"
```

Comment:
nope, still doesn't work. installs dev-master, at least with `~`, do you think it will work differently with `^` ?